src/credential/issuance/06-obtain-credential.ts

```diff
@@ -0,0 +1,141 @@
+import * as z from "zod";
+import uuid from "react-native-uuid";
+import {
+  type CryptoContext,
+  verify as verifyJwt,
+  decode as decodeJwt,
+} from "@pagopa/io-react-native-jwt";
+import { createDPopToken } from "../../utils/dpop";
+import { createNonceProof } from "../../utils/par";
+import type { StartFlow } from "./01-start-flow";
+import { hasStatus, type Out } from "../../utils/misc";
+import type { EvaluateIssuerTrust } from "./02-evaluate-issuer-trust";
+import type { AuthorizeAccess } from "./05-authorize-access";
+import { SdJwt4VC } from "../../sd-jwt/types";
+
+const CredentialEndpointResponse = z.object({
+  credential: z.string(),
+  format: z.literal("vc+sd-jwt"),
+  /* YAGNI
+  Ideally, we may request more than one credential after a single authorization phase
+  Each new call to the credential endpoint must carry a different nonce value,
+  otherwise it would be rejected.
+  So every response has a new nonce value to eventually be used in the next request.
+  It's not implementd so far
+  c_nonce: z.string(),
+  c_nonce_expires_in: z.number().positive(), 
```

Comment:
TODO?

Reply:
We don't really need commented code. Removed in https://github.com/pagopa/io-react-native-wallet/pull/62/commits/32063c845f6fbc0b5410eda0904733319569ca16